apps/betterangels-backend/hmis/api_bridge.py

```diff
@@ -62,8 +64,16 @@ def _make_request(self, body: dict[str, Any], timeout: Optional[float] = None) -
             json=body,
             timeout=timeout,
         )
+
+        response = resp.json() or {}
+
+        if errors := response.get("errors"):
+            if is_hmis_unauthenticated(errors):
+                # TODO: destroy session here?
```

Comment:
Yah if we get an unauthenticated error we'd want to reauth. Though this is likely an edge case since the session ttls should match.